bridge/engine/EngineMixer.cpp

```diff
@@ -1318,6 +1322,12 @@ void EngineMixer::sendMessagesToNewDataStreams()
             continue;
         }
 
+        if (!isDominantSpeakerMessageBuild)
+        {
+            _activeMediaList->makeDominantSpeakerMessage(dominantSpeakerMessage);
```

Comment:
Minor improvement,  build a message only there is at least 1 participant that should receive it